static/app/gettingStartedDocs/react-native/react-native.tsx

```diff
@@ -72,6 +74,30 @@ shopCheckout() {
   }
 }`;
 
+const getReplaySetupSnippet = (params: Params) => `
+import * as Sentry from '@sentry/react-native';
+
+Sentry.init({
+  dsn = "${params.dsn.public}",
+  _experiments: {
+    replaysSessionSampleRate: 1.0,
+    replaysOnErrorSampleRate: 1.0,
+  },
+  integrations: [
+    Sentry.mobileReplayIntegration({
+      maskAllText: true,
+      maskAllImages: true,
+    }),
```

Comment:
These values are repeated in the configuration snippet, I would leave them out here since they are not needed and will make the init snipped a bit less overwhelming.

Also it will match the Browser JS which also by default doesn't show any config options.

```suggestion
    Sentry.mobileReplayIntegration(),
```

Reply:
oh yes, good point -- i copied this from the docs so maybe we want to update it there too? 

https://docs.sentry.io/platforms/react-native/session-replay/

Reply:
fixed here https://github.com/getsentry/sentry-docs/pull/11192